readme.txt

```diff
@@ -18,9 +18,7 @@ Whether you’re building a single-page website, a blog, a business website, or
 == Changelog ==
 
 = 1.0 =
-* Released: December
-
-https://wordpress.org/support/article/twenty-twenty-two-changelog#Version_1.0
+* Released: January 25, 2022
```

Comment:
I also updated this while I was in here. The release date was wrong, and that link doesn't exist.